packages/lib-classifier/webpack.dist.js

```diff
@@ -1,19 +1,23 @@
 const path = require('path')
 const webpack = require('webpack')
-const WorkboxPlugin = require('workbox-webpack-plugin')
+const PeerDepsExternalsPlugin = require('peer-deps-externals-webpack-plugin')
 
 const EnvironmentWebpackPlugin = new webpack.EnvironmentPlugin({
   DEBUG: false,
   NODE_ENV: 'production',
   PANOPTES_ENV: 'production'
 })
 
-const WorkboxPluginConfig = new WorkboxPlugin.InjectManifest({
-  swSrc: './src/workers/queue.js'
-})
-
+// The peer deps external plugin doesn't work with scoped packages
+// so we add them explicitly to webpack's externals config
+// see: https://github.com/Updater/peer-deps-externals-webpack-plugin/issues/5
 module.exports = {
   entry: './src/components/Classifier/index.js',
+  externals: [
+    /^(@zooniverse\/grommet-theme\/.*)$/i,
```

Comment:
We could have one regex for everything under the `@zooniverse` namespace

Reply:
We could, though the issue workaround showed specific regexes to match namespaced packages, so I just followed that. I'm not sure if there's a reason for that.